app/grandchallenge/reader_studies/migrations/0009_auto_20200316_1439.py

```diff
@@ -0,0 +1,31 @@
+# Generated by Django 3.0.2 on 2020-03-16 14:39
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("reader_studies", "0008_auto_20200123_0931"),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name="question",
+            name="answer_type",
+            field=models.CharField(
+                choices=[
+                    ("STXT", "Single line text"),
+                    ("MTXT", "Multi line text"),
+                    ("BOOL", "Bool"),
+                    ("HEAD", "Heading"),
+                    ("2DBB", "2D bounding box"),
+                    ("DIST", "Distance measurement"),
+                    ("MDIS", "Multiple distance measurements"),
```

Comment:
We don't want individual points? I can think of some studies in the past that have needed this.